python_modules/dagster/dagster/_cli/api.py

```diff
@@ -650,6 +650,18 @@ def grpc_command(
             "empty_working_directory",
         ]
     ):
+        # in the grpc api CLI we never load more than one module at a time
+        maybe_module = kwargs["module_name"]
```

Comment:
You could also enforce this at the click level instead by passing a flag into python_target_click_options depending on the callsite, but that might be a pain

Reply:
Yeah I think this simpler